### ISSUE TITLE
operator nan_to_num to ONNX

### ISSUE THREAD

**Issue by iamlockelightning** [pytorch/pytorch#70886]
### 🚀 The feature, motivation and pitch

Get "RuntimeError: Exporting the operator nan_to_num to ONNX opset version 9 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub." when using `torch.nan_to_num()`

```python
import torch
import pytorch_lightning as pl

class SimpleModel(pl.LightningModule):
    def __init__(self):
        super().__init__()
        self.l1 = torch.nn.Linear(in_features=64, out_features=4)

    def forward(self, x):
        y = self.l1(x.view(x.size(0), -1))
        y = torch.nan_to_num(y)
        return y

filepath = "model.onnx"
model = SimpleModel()
input_sample = torch.randn((1, 64))
model.to_onnx(filepath, input_sample, export_params=True)
```

Opened an issue in the ONNX repo(https://github.com/onnx/onnx/issues/3917) and recommended opening one here.

My environment is:
```
pytorch-lightning       1.5.7
torch                   1.10.1
torchaudio              0.10.1
torchmetrics            0.6.2
torchvision             0.11.2
onnx                    1.10.2
onnxruntime             1.9.0
```

**Comment by Julius-Plehn** [comment]
I have an urgent need for this feature. Is there any way I could help with the implementation or a possibility to work around this case in the meantime? Thanks in advance!

**Comment by shubhambhokare1** [comment]
PR #72090 should add support for this operator. Let me know if this resolves your issue.

**Comment by iamlockelightning** [comment]
> PR #72090 should add support for this operator. Let me know if this resolves your issue.

Thanks, @shubhambhokare1. It looks awesome!

**Comment by olegggatttor** [comment]
Hi, @iamlockelightning! Which PyTorch version should I use to access this feature?

**Comment by garymm** [comment]
Fix merged into master in #73103.
@olegggatttor PyTorch nightly. Should be released in 1.12.
